source/fundamentals/stable-api.txt

```diff
@@ -51,10 +51,10 @@ version of the {+stable-api+}.
 
 .. tip::
 
-   If you need to run commands using more than one version of the 
+   If you must run commands using more than one version of the 
    {+stable-api+}, instantiate a separate client with that version.
 
-   If you need to run commands not covered by the {+stable-api+}, make sure the
+   If you must run commands not covered by the {+stable-api+}, make sure the
```

Comment:
S: You could make a change here similar to the ones you made previously: 

"To run commands that use more than one version..."